src/core/lib/transport/metadata_batch.cc

```diff
@@ -204,7 +204,7 @@ StaticSlice HttpMethodMetadata::Encode(ValueType x) {
     case kGet:
       return StaticSlice::FromStaticString("GET");
     default:
-      abort();
+      return StaticSlice::FromStaticString("???");
```

Comment:
Maybe this should say "Invalid value" or something.

As per our conversation, please add a TODO about splitting up the debug string and wire format string representations.